appengine-java8/endpoints-v2-skeleton/pom.xml

```diff
@@ -0,0 +1,124 @@
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
+
+    <!-- [START pom_dependencies] -->
+    <dependencies>
+        <!-- Compile/runtime dependencies -->
+        <dependency>
+            <groupId>com.google.endpoints</groupId>
+            <artifactId>endpoints-framework</artifactId>
+            <version>2.0.9</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.appengine</groupId>
+            <artifactId>appengine-api-1.0-sdk</artifactId>
+            <version>1.9.59</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+            <version>1</version>
+        </dependency>
+    </dependencies>
+    <!-- [END pom_dependencies] -->
+
+    <!-- [START pom_build] -->
+    <build>
+        <!-- for hot reload of the web application-->
+        <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes</outputDirectory>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <webResources>
+                        <resources>
+                            <directory>${basedir}/src/main/webapp/WEB-INF</directory>
+                            <filtering>true</filtering>
+                            <targetPath>WEB-INF</targetPath>
+                        </resources>
+                    </webResources>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>appengine-maven-plugin</artifactId>
+                <version>1.3.2</version>
+                <configuration>
+                    <!-- deploy configuration -->
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>endpoints-framework-maven-plugin</artifactId>
+                <version>1.0.2</version>
+                <configuration>
+                    <!-- plugin configuration -->
+                    <hostname>YOUR_PROJECT_ID.appspot.com</hostname>
+                </configuration>
+            </plugin>
+            <!--
+                 The versions-maven-plugin is used for dependency updates and
+                 is not required.
+            -->
+            <plugin>
```

Comment:
If this is not required and we cal this a skeleton app, maybe we can opt out of including it ? (making for a simpler `pom.xml`)

Reply:
I guess I misunderstood, but I thought Dependency Auto-update used this plugin to know which dependencies to update?

Reply:
we don't include it within each sample though:
already included here.
https://github.com/GoogleCloudPlatform/java-repo-tools/blob/master/pom.xml#L237

Reply:
ah thanks for clarifying! Will fix and then merge.